lib/combobox.js

```diff
@@ -388,6 +376,7 @@ module.exports = React.createClass({
   },
 
   render: function() {
+
```

Comment:
We should respect the formatting of the original library to prevent future merge conflicts